src/style.css

```diff
@@ -1,3 +1,78 @@
+:root {
+  --blue: #00acdc;
+  --black: #000;
+}
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+}
+
+.header__logo {
+  height: 60px;
+}
+
+.header__img {
+  margin: 10px 0;
+}
+
+.nav {
+  height: 60px;
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  list-style: none;
+  padding-left: 0;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  display: block;
+  color: var(--black);
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 60px;
```

Comment:
set a correct figma line-height value and use padding

Reply:
The same

Reply:
isn't fixed. 
You need to set padding to your links (it will be top and bottom padding) and set correct line-height value from figma